rebasehelper/logger.py

```diff
@@ -178,19 +112,90 @@ def emit(self, record):
+                           formatter_object: logging.Formatter = None) -> ColorizingStreamHandler:
+        """Adds stream handler to the given logger.
+
+        Args:
+            logger_object: Logger object to add the handler to.
+            level: Severity threshold.
+            formatter_object: Formatter object used to format logged messages.
+
+        Returns:
+            Created stream handler instance.
+
+        """
+        console_handler = ColorizingStreamHandler()
+        if level:
+            console_handler.setLevel(level)
+        if formatter_object:
+            console_handler.setFormatter(formatter_object)
+        logger_object.addHandler(console_handler)
+        return console_handler
+
+    @staticmethod
+    def add_file_handler(logger_object: logging.Logger, path: str, formatter_object: logging.Formatter = None,
+                         level: int = None) -> None:
+        """Adds file handler to the given logger.
+
+        Args:
+            logger_object: Logger object to add the handler to.
+            path: Path to a log file.
+            formatter_object: Formatter object used to format logged messages.
+            level: Severity threshold.
+
+        Returns:
+            Created file handler instance.
+
+        """
+        try:
+            file_handler = logging.FileHandler(path, 'w')
+            if level:
+                file_handler.setLevel(level)
+            if formatter_object:
+                file_handler.setFormatter(formatter_object)
+            logger_object.addHandler(file_handler)
+        except (IOError, OSError):
+            logger_object.warning('Can not create log in %s', path)
+
+
+def create_file_handlers(results_dir: str) -> None:
+    """Creates rebase-helper file handlers.
+
+    Args:
+        results_dir: Path to rebase-helper-results directory.
+
+    Returns:
+        Path to debug log file.
+
+    """
+    logs_dir = os.path.join(results_dir, constants.LOGS_DIR)
+    logger = logging.getLogger("rebasehelper")
+
+    log_formatter = logging.Formatter("%(message)s")
+    debug_log_formatter = logging.Formatter("%(asctime)s %(filename)s:%(lineno)s %(funcName)s: %(message)s")
+
+    debug_log = os.path.join(logs_dir, constants.DEBUG_LOG)
+    LoggerHelper.add_file_handler(logger, debug_log, debug_log_formatter, logging.DEBUG)
+    verbose_log = os.path.join(logs_dir, constants.VERBOSE_LOG)
+    LoggerHelper.add_file_handler(logger, verbose_log, log_formatter, CustomLogger.VERBOSE)
+    info_log = os.path.join(logs_dir, constants.INFO_LOG)
+    LoggerHelper.add_file_handler(logger, info_log, log_formatter, logging.INFO)
+
+
+def create_stream_handlers() -> Tuple[ColorizingStreamHandler, ColorizingStreamHandler]:
+    logger = logging.getLogger('rebasehelper')
+    formatter = logging.Formatter("%(levelname)s: %(message)s")
+    main = LoggerHelper.add_stream_handler(logger, logging.INFO, formatter)
```

Comment:
Just a nitpick, but the stream handlers don't have to be named. They are being used as a list anyway.

Reply:
Ok, the first one is special, but you could still return a list instead of a tuple.

Reply:
As you already said, the first one is used to set verbostiy. I think that with only 2 handlers, using tuple and naming them is a bit more clear than using a list.